pkgs/os-specific/linux/ledger-udev-rules/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ledger-udev-rules-unstable-${version}";
```

Comment:
The point of using pname is that you do not append the version to it. That gets done automatically.

Reply:
Should I leave the `unstable-` part then?

Reply:
I think you should have
```
pname = "ledger-udev-rules";
version = "unstable-2019-02-13";
```

Reply:
> If a package is not a release but a commit from a repository, then the version part of the name must be the **date of that (fetched) commit**. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

From the manual.

Reply:
What @nyanloutre says suggests makes the most sense to me. Say they release a version of the udev rules called "1.0.0" then you'd want to change version to "1.0.0" and drop the unstable part. Having it like that would only change one line.

Reply:
Got it. There are packages that add `unstable` to their version though, so probably the manual needs to be made clearer/fixed.